OperatingSystem-Services/Platform-Linux/Identity Management/KeyCloak/KeyCloak.md

```diff
@@ -4,38 +4,38 @@ Keycloak is an open-source identity and access management (IAM) solution designe
 Official Website: https://www.keycloak.org/
 
 ## Getting Started (Docker)
-In order to get started we will need to install docker as this documentation focuses on using the advantages of docker to better use KeyCloak in general. If you already have docker installed you can skip to KeyCloak Install.
+In order to get started we will need to install Docker as this documentation focuses on using the advantages of Docker to better use KeyCloak in general. If you already have Docker installed you can skip to KeyCloak Install.
 
-Below is the squential command lines needed to download and install docker, more in depth documentation on the workings of docker should be found elsewhere on this repo.
+Below is the squential command lines needed to download and install Docker, more in depth documentation on the workings of Docker should be found elsewhere on this repo.
 
 ```sh
 sudo apt update
 sudo apt install apt-transport-https ca-certificates curl software-properties-common
-curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
-sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+curl -fsSL https://download.Docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo add-apt-repository "deb [arch=amd64] https://download.Docker.com/linux/ubuntu $(lsb_release -cs) stable"
 sudo apt update
-sudo apt install docker-ce
+sudo apt install Docker-ce
```

Comment:
I am not sure if this is fine; check if package names are case-sensitive. As far as I am aware they may not be.

Reply:
Same this elsewhere throughout